### ISSUE TITLE
Play button - focus order (WCAG 2.4.3)

### ISSUE THREAD

**Issue by HendrikWismans** [Splidejs/splide-extension-video#50]
### Checks

- [X] Not a duplicate.
- [X] Not a question, feature request, or anything other than a bug report directly related to Splide. Use Discussions for these topics: https://github.com/Splidejs/splide/discussions

### Version

4.0.2

### Description

When i'm using my keyboard (no mouse) the focus order after the play button is incorrect. When i hit tab, focus should go to the video. But the focus goes to the pagination. When i press shift-tab, my focus comes into the video iframe.

This is a WCAG 2.4.3 failure and can easily fixed when html order is changed.

### Reproduction Link

https://splidejs.com/extensions/video/

### Steps to Reproduce

1. https://splidejs.com/extensions/video/
2. Use keyboard **tab** and go to the play button
3. press **enter** or **space** (video plays)
4. press **tab** and focus goes to pagination.
5. **shift tab** goes to video

### Expected Behaviour

The button.splide__video__play is renderer after div.splide__video__wrapper. If button.splide__video__play is renderer before div.splide__video__wrapper focus order is correct.

**Comment by NaotoshiFujita** [comment]
Fixed in the latest version.

**Comment by HendrikWismans** [comment]
Thanks!
